pkg/apis/serving/v1alpha1/revision_types.go

```diff
@@ -120,9 +130,18 @@ type RevisionSpec struct {
 	// ConcurrencyModel specifies the desired concurrency model
 	// (Single or Multi) for the
 	// Revision. Defaults to Multi.
+	// Deprecated in favor of ContainerConcurrency.
```

Comment:
I'd like to see this be _deprecated_, not discontinued, at least until we have some API versioning in place:

> In several fields, deprecation is the discouragement of use of some terminology, feature, design, or practice, typically because it has been superseded or is no longer considered efficient or safe, **without completely removing it or prohibiting its use**. It can also imply that a feature, design, or practice will be removed or discontinued entirely in the future
([wikipedia](https://en.wikipedia.org/wiki/Deprecation))

Reply:
FWIW: I manually tried this out and it works as expected:

1. No concurrencyModel specified results in a concurrency=0 passed
2. concurrencyModel=Multi results in a concurrency=0 passed
3. concurrencyModel=Single results in a concurrency=1 passed